### ISSUE TITLE
Can you support Chinese?

### ISSUE THREAD

**Issue by xp12047** [yashs662/rust_kanban#4]
I noticed that it doesn't seem to support Chinese characters.

**Comment by yashs662** [comment]
Yes, currently only English Characters are supported. Support for more languages will be coming in future releases!!! If you wish to implement it your self feel free to create a pull request✌️

**Comment by xp12047** [comment]
> 是的，目前只支持英文字符。未来版本将支持更多语言！！！如果你想实现它，你可以随意创建一个 pull request✌️

I am looking forward to supporting Chinese in the future.   
Sorry, I can't program.  I tried to learn but couldn't stick to it.
